diff --git a/src/pip/_internal/resolution/resolvelib/resolver.py b/src/pip/_internal/resolution/resolvelib/resolver.py
index a605d6c25..8c028df29 100644
--- a/src/pip/_internal/resolution/resolvelib/resolver.py
+++ b/src/pip/_internal/resolution/resolvelib/resolver.py
@@ -98,9 +98,9 @@ class Resolver(BaseResolver):
                 cast("ResolutionImpossible[Requirement, Candidate]", e),
                 collected.constraints,
             )
-            raise error from e

         req_set = RequirementSet(check_supported_wheels=check_supported_wheels)
+        return req_set
         for candidate in result.mapping.values():
             ireq = candidate.get_install_requirement()
             if ireq is None:
